src/operators/fsApi/setPageProperties.ts

```diff
@@ -0,0 +1,17 @@
+import FSApiOperator from './fsApi';
+
+export default class SetPagePropertiesOperator extends FSApiOperator {
+  // eslint-disable-next-line class-methods-use-this
+  prepareData(inputData:any[]): any[] | null {
+    if (inputData === null || inputData.length < 1) {
```

Comment:
Should we throw an `Error`?  The operator expects inputData, right?  Is there a scenario where not receiving data but expecting it to be sent to `setProperties` is OK?  Returning `null` is typically used to halt the operator chain under expected conditions - like when a query doesn't match and the chain is expected to stop.